draft-ietf-oauth-device-flow.xml

```diff
@@ -649,11 +654,12 @@
   </t>
 
   <t>
-  The assumption of this specification is that the secondary device the
+  The assumption of this specification is that the separate device the
   user is authorizing the request on does not have a way to communicate back
-  to the OAuth client. Only a one-way channel is required to make this flow
-  useful in many scenarios. For example, an HTML application on a TV that can
-  only make outbound requests. If a return channel were to exist for the chosen user
+  to device with the OAuth client. This protocol only requires a one-way
```

Comment:
I suspect that the wording "communicate back to device with the OAuth client" needs to be corrected.

Reply:
Ah, missed this comment. Have staged a change for the next release it now reads "communicate back to the device with the OAuth client"